features/configurator.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Define global parameters
+
+  Scenario: Throws exception for an invalid config spec
+    Given an empty directory
+    And a wp-invalid-config-spec.php file:
+      """
+      <?php
+      // A hacky way to inject our constant into the runtime.
+      define( 'WP_CLI_CONFIG_SPEC_PATH', 'invalid-path' );
+      
+      $wp_cli_info = json_decode( trim( shell_exec( 'wp cli info --format=json' ) ), true );
+      require $wp_cli_info['wp_cli_dir_path'] . '/php/boot-fs.php';
```

Comment:
Worth noting: this approach to testing won't work when WP-CLI is run against a Phar.

With a Phar, I believe we can simply `require` the executable directly.

Reply:
So this won't work in https://github.com/wp-cli/automated-tests where nighty phar is used for testing.

Reply:
> So this won't work in https://github.com/wp-cli/automated-tests where nighty phar is used for testing.

I think this is correct, yes.

I'll figure out some mechanism where it handles both contexts.